programs/rewards/tests/rewards/distribute_rewards.rs

```diff
@@ -31,7 +31,7 @@ async fn setup() -> (ProgramTestContext, TestRewards, Pubkey) {
         &mut context,
         &test_rewards.token_mint_pubkey,
         &rewarder.pubkey(),
-        100,
+        150,
```

Comment:
I haven't checked the code above, tbh, but I'm curious why you changed the value